src/components/about/HISS.jsx

```diff
@@ -1,7 +1,49 @@
 import React from "react";
+import Header from "../Header.jsx"; // previously was: src/components/Header
+import Image from "next/image";
+import About from "../../../public/images/about.png";
 
 const HISS = () => {
-  return <div>HISS</div>;
+  return (
+    <div className="flex-col items-center justify-center ml-1 inline-flex">
```

Comment:
1. why ml-1?
2. add fraction based width, this should allow you to move all of the breaks